docs/_reference/rtm-api.md

```diff
@@ -31,6 +32,13 @@ slug: rtm-api
 <td align="center">✗</td>
 <td></td>
 </tr>
+<tr>
+<td align="center">webClient</td>
+<td align="center"><code><a href="https://github.com/slackapi/node-slack-sdk/blob/main/docs/_reference/web-api.md#webclient" title="">WebClient</a></code></td>
+<td align="center">✗</td>
+<td align="center">An optional parameter to provide a customized <a href="https://github.com/slackapi/node-slack-sdk/blob/main/docs/_reference/web-api.md#webclient">WebClient</a>. Any desired options for the custom client must be set in this parameter (<code>webClient</code>) as they will take precedence over other arguments passed into <code>RTMClient</code>.</td>
```

Comment:
```suggestion
<td align="center">An optional parameter to provide a customized <a href="web-api#webclient">WebClient</a>. Any desired options for the custom client must be set in this parameter (<code>webClient</code>) as they will take precedence over other arguments passed into <code>RTMClient</code>.</td>
```

Link to hosted docs instead of github